<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>actualizarProducto</class>
 <widget class="QDialog" name="actualizarProducto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font-family:century gothic;
	font-size:24px;
}

QPushButton{
	background:#0069BF;
	border-radius:10px;
	color:white;
}

QPushButton:hover{
	background:#008EFF;
}

QLineEdit{
	border-radius:14px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Ingresa ID del producto a modificar:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Marca:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leMarca"/>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Modelo:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leModelo"/>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#0069BF;
color: white;
</string>
     </property>
     <property name="text">
      <string>Actualizar Producto</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Precio:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lePrecio"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Ingresa Nueva Info:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Descripcion:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leDescripcion"/>
     </item>
    </layout>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QPushButton" name="btnActualizar">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>Actualizar</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>ID Producto:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
